packages/vscode-apollo/README.md

```diff
@@ -1,18 +1,154 @@
+
+Client side schema definitions can be spread throughout the client app project and will be merged together to create one single schema. This can be controlled through the `apollo.config.js` at the root of he project:
+
+```js
+module.exports = {
+  client: {
+    service: "my-graphql-app"
+    includes: ["./src/**/*.js"],
+    excludes: ["**/__tests__/**"]
+  }
+}
+```
+
+By default, the VS Code extension will look for all files under "./src" to find both the operations and schema definitions for building a complete schema for the application.
+
+<h2 id="features">Features</h2>
+
+Apollo for VS Code brings many helpful features for working on a GraphQL project.
+
+<h3 id="syntax">Syntax highlighting</h3>
+
+Apollo's editor extension provides syntax highlighting for all things GraphQL, including schema definitions in `.graphql` files, complex queries in TypeScript, and even client-only schema extensions. Syntax highlighting for GraphQL works out-of-the-box for `.graphql`, `.gql`, `.js` and `.ts` file types!
+
+<h3 id="autocomplete">Intelligent autocomplete</h3>
+
+Once configured, editors have full knowledge of the schema clients are running operations against, including client-only schemas (for things like local state mutations). Because of this, editors have the ability to autocomplete fields and arguments as you type.
+
+<img src="./images/marketplace/autocomplete.gif" width="80%" style="margin: 5%" alt="vscode completing a field when typing">
+
+<h3 id="errors-and-warnings">Inline errors and warnings</h3>
+
+Editors can use local or published schemas to validate operations before running them. **Syntax errors**, **invalid fields or arguments**, and even **deprecated fields** instantly appear as errors or warnings right in your editor, ensuring all developers are working with the most up-to-date production schemas.
+
+<img src="./images/marketplace/warnings-and-errors.gif" width="80%" style="margin: 5%" alt="tooltip showing a field deprecation warning and error">
+
+<h3 id="field-type-info">Inline field type information</h3>
+
+Because of GraphQL's strongly-typed schema, editors not only know about which fields and arguments are valid, but also what types are expected. Hover over any type in a valid GraphQL operation to see what type that field returns and whether or not it can be null.
+
+<img src="./images/marketplace/type-info.png" width="80%" style="margin: 5%" alt="a tooltip showing a Boolean type for a field">
+
+<h3 id="performance-insights">Performance insights</h3>
+
+GraphQL operations provide incredible flexibilty in what data is requested from a service. This can sometimes lead to unknown performance characteristics of how an operation will run. Thanks to the trace wharehouse in the Apollo GraphQL Platform, teams no longer will be surprised by how long an operation takes.
```

Comment:
```suggestion
GraphQL operations provide incredible flexibilty in what data is requested from a service. This can sometimes lead to unknown performance characteristics of how an operation will run. Thanks to the trace warehouse in the Apollo GraphQL Platform, teams will no longer be surprised by how long an operation takes.
```